test/integration.test.ts

```diff
@@ -0,0 +1,16 @@
+import {fetchBeacon, fetchBeaconByTime, HttpCachingChain, HttpChainClient} from '../lib';
+import 'jest-fetch-mock'
+
+describe('randomness client', () => {
+    const chain = new HttpCachingChain('https://api.drand.sh')
```

Comment:
I'd argue tests should be running against the testnet :P

Reply:
good point! was testing something against mainnet when I wrote these